packages/core/src/components/slider/multiSlider.tsx

```diff
@@ -42,9 +42,15 @@ export interface ISliderBaseProps extends IProps, IIntentProps {
 
     /**
      * Increment between successive labels. Must be greater than zero.
-     * @default 1
+     * @default inferred 1
+     */
+    labelStepSize?: number | null;
```

Comment:
as you just suggested, I would like this to use `undefined` instead of `null`. there's no need to introduce _two_ kinds of "non-values" unless absolutely necessary.

```suggestion
    labelStepSize?: number;
```